### PR TITLE
implement CheckSecondaryLockStatus

### PR DESCRIPTION
**Pull request by sticnarf**

### CODE REVIEW

**Comment by sticnarf** [comment]
@cfzjywxk @coocood PTAL

File: tikv/mvcc.go [hunk 1]
```diff
@@ -451,6 +451,55 @@ func (store *MVCCStore) CheckTxnStatus(reqCtx *requestCtx,
...
+					Key:             key,
+					LockTtl:         uint64(lock.TTL),
+					LockType:        kvrpcpb.Op(lock.Op),
+					LockForUpdateTs: lock.ForUpdateTS,
+					UseAsyncCommit:  lock.UseAsyncCommit,
+					MinCommitTs:     lock.MinCommitTS,
+				})
+			}
+		} else {
+			commitTS, err := store.checkCommitted(reqCtx.getDBReader(), key, startTS)
```
Comment (coocood) [inline tikv/mvcc.go:489]: May be the key is already rolled back, we should call `checkExtraTxnStatus`.

**Comment by sticnarf** [comment]
FIXME: This PR forgets to wake up rolled back locks.

File: tikv/mvcc.go (hunk 1 above)
Comment (sticnarf) [reply tikv/mvcc.go:495]: I've updated the PR and add the Op_Lock test case. PTAL again.

**Comment by coocood** [comment]
LGTM

**Comment by coocood** [comment]
@cfzjywxk PTAL

**Comment by cfzjywxk** [comment]
LGTM

**Pull request merged**